projects/fsastorefrontstyles/scss/base/comparison-table.scss

```diff
@@ -132,4 +131,35 @@
       border-left: none;
     }
   }
+  .ribbon {
+    position: relative;
+    width: calc(100% + 20px);
+    left: -10px;
+    top: 10px;
+    box-shadow: 0 0 0 2px var(--cx-color-primary),
+      0px 21px 5px -18px rgb(0 0 0 / 60%);
+    border-top: 1px dashed;
+    border-bottom: 1px dashed;
+    padding: 5px 0;
+    background-color: var(--cx-color-primary);
+    font-size: 10px;
```

Comment:
Since we don't have 10px for font-size defined, maybe use calc to set it relative to our base font size.
`font-size: calc(var(--cx-xs-font-size) * 0.95);`